perses/tests/test_fah.py

```diff
@@ -89,7 +93,7 @@ def test_pipeline_protein():
     # need to replace ligand and protein location in file
 
     setup_options['phase_project_ids'] = {'complex':'temp-complex','apo':'temp-apo'}
-    setup_options['protein_kwargs']['ligand_file'] = ligand_file
+    setup_options['protein_kwargs']['ligand_input'] = ligand_file
```

Comment:
Is this fixing a bug or making something more consistent? Is this how we handle it in [fah_generator.py](https://github.com/choderalab/perses/blob/main/perses/app/fah_generator.py)?

Reply:
Same as above: This is fixing a bug for running protein mutations. The PointMutationExecutor API changed awhile ago, but fah_generator.py was not updated.